dev-packages/application-manager/src/application-package-manager.ts

```diff
@@ -68,11 +69,11 @@ export class ApplicationPackageManager {
     }
 
     async startElectron(args: string[]): Promise<void> {
-        return this.__process.bunyan(
-            this.__process.spawnBin('electron', [this.pck.frontend('electron-main.js'), ...args], {
-                stdio: [0, 'pipe', 'pipe']
-            })
-        );
+        const child = this.__process.spawnBin('electron', [this.pck.frontend('electron-main.js'), ...args], {
+            stdio: [0, 'pipe', 'pipe']
+        });
+        child.stdout.pipe(process.stdout);
```

Comment:
The first arg (0) looks odd. At least I cannot see it being specified here:
https://nodejs.org/api/child_process.html#child_process_options_stdio

Otherwise, I would interpret the `stdio` option to do exactly the piping you do manually afterward. @akosyakov Do you have an opinion on this?

Reply:
it is a shorter way for `[process.stdin, 'pipe', 'pipe']` - from https://nodejs.org/api/child_process.html#child_process_options_stdio:
`'inherit' - equivalent to [process.stdin, process.stdout, process.stderr] or [0,1,2]`

Reply:
I don't think we need piping anymore, it was because of `bunyan`:
```ts
return this.__process.spawnBin('electron', [this.pck.frontend('electron-main.js'), ...args]);
```

Reply:
:+1: It works perfectly.